src/flag_gems/utils/libentry.py

```diff
@@ -122,7 +122,11 @@ def run(self, *args, **kwargs):
                         raise RuntimeError("Invalid Runtime Function")
                     fn = fn.fn
                 for p in self.jit_function.params:
-                    if p.is_constexpr and p.name not in constexprs:
+                    if (
+                        p.is_constexpr
+                        and p.name not in constexprs
+                        and (p.default is not inspect._empty)
+                    ):
```

Comment:
Did we trigger any error because of this?

Reply:
yes. when we pass constexpr as a key-word argument, it will be covered by the default empty in metadata, which causes error in grid function.